scripts/west_commands/runners/linkserver.py

```diff
@@ -186,12 +186,7 @@ def _build_override_cli(self):
 
     def flash(self, **kwargs):
 
-        if self.core is not None:
-            _cmd_core = ":"+self.core
-        else:
-            _cmd_core = ""
-
-        linkserver_cmd = ([self.linkserver, "flash"] + ["--probe", str(self.probe)] + self.override_cli + [self.device+_cmd_core])
+        linkserver_cmd = ([self.linkserver, "flash"] + ["--probe", str(self.probe)] + self.override_cli + [self.device])
```

Comment:
Could it be conditional?  Something like self.linkserver_version_str >= "v1.5.30"
So the runner can work for all LinkServer versions.

Reply:
No, because the newer version just enforces the core be not specified. In the past, it ignored it.